src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/model/PrisonerDetail.kt

```diff
@@ -52,10 +52,6 @@ data class PrisonerDetail(
   @Column(name = "OFFENCES")
   val offences: String? = null,
 
-  @Lob
-  @Column(name = "OFFENCES_IN_CUSTODY")
-  val offencesInCustody: String? = null,
```

Comment:
not needed since we actually grab the adjudications and then call them `offencesInCustody` 🤯